clients/src/main/resources/common/message/FetchResponse.json

```diff
@@ -102,6 +104,15 @@
           "about": "The preferred read replica for the consumer to use on its next fetch request"},
         { "name": "Records", "type": "records", "versions": "0+", "nullableVersions": "0+", "about": "The record data."}
       ]}
+    ]},
+    { "name": "NodeEndpoints", "type": "[]NodeEndpoint", "versions": "16+", "taggedVersions": "16+", "tag": 0,
```

Comment:
let's not reuse the tags as I mentioned in the other PR.

Reply:
I replied to this in the other PR https://github.com/apache/kafka/pull/14444#discussion_r1367116597

Reply:
> it looks like tags are scoped to the list level so this isn't really the same tag. They also need to be contiguous within their scope so this gives an error if I try to tag NodeEndpoints to something other than 0

Sharing the comment here for clarity.